views/includes/metaFooterJS.njk

```diff
@@ -1,9 +1,10 @@
-<script src="https://www.google-analytics.com/cx/api.js"></script>
 <script src="{{ 'javascripts/app.js' | getCachebustedPath }}"
         id="js-script-main"
         data-ga-code="{{ appData.config.get('googleAnalyticsCode') }}"{% if ab %}
         data-ab-id="{{ ab.id }}"
         data-ab-variant="{{ ab.variantId }}"{% endif %}></script>
 
+
+<script async src="https://www.google-analytics.com/cx/api.js"></script>
```

Comment:
Does this work okay like this with the A/B tests?

Reply:
Hmm, good point, will check.